pkg/apis/apps/types.go

```diff
@@ -100,12 +89,11 @@ const (
 	OnDeleteStatefulSetStrategyType = "OnDelete"
 )
 
-// PartitionStatefulSetStrategy contains the parameters used with the
-// PartitionStatefulSetStrategyType.
-type PartitionStatefulSetStrategy struct {
-	// Ordinal indicates the ordinal at which the StatefulSet should be
+// RollingUpdateStatefulSetStrategy is used to communicate parameter for RollingUpdateStatefulSetStrategyType.
+type RollingUpdateStatefulSetStrategy struct {
+	// Partition indicates the ordinal at which the StatefulSet should be
 	// partitioned.
-	Ordinal int32
+	Partition int32 `json:"ordinal" protobuf:"varint,1,opt,name=ordinal"`
```

Comment:
I think until we add a new field, it's OK. Ultimately, I feel there is difference between 0 and nil (0 shouldn't update the status to signal a complete update but nil should so we can have post-hooks).